libs/wingsdk/src/shared-azure/function.inflight.ts

```diff
@@ -0,0 +1,37 @@
+import { IFunctionClient } from "../cloud";
+import { Util as http } from "../http";
+import { Trace } from "../std";
+
+export class FunctionClient implements IFunctionClient {
+  constructor(private functionName: string) {}
+
+  /**
+   * Invoke the function, passing the given payload as an argument.
+   *  @returns the function returned payload only
+   */
+  public async invoke(payload: string): Promise<string> {
+    try {
+      const res = await http.post(
+        `https://${this.functionName}.azurewebsites.net/api/${this.functionName}`,
+        { body: payload }
+      );
+      return res.body ?? "";
+    } catch (error) {
+      throw new Error(
+        `Error while invoking the function ${this.functionName}:\n${
+          (error as Error).message
+        }`
+      );
+    }
+  }
+
+  /**
+   * Invoke the function, passing the given payload as an argument.
+   *
+   * @returns the function returned payload and logs
+   */
+  public async invokeWithLogs(payload: string): Promise<[string, Trace[]]> {
+    console.error("invokeWithLogs is not implemented yet on tf-azure");
+    return [await this.invoke(payload), []];
+  }
```

Comment:
I think this method should throw an error instead of logging one. `TestRunner` for `tf-azure` can use `invoke` instead of `invokeWithLogs` if this method isn't supported

Reply:
I did it to be able to use the same function between all tf targets- I'll add an issue to include traces in tf-azure